src/navigationDrawer/navigationDrawer.component.tsx

```diff
@@ -66,7 +66,10 @@ const LinkListItem = (props: LinkListItemProps): React.ReactElement => (
   // an `any` is required here to pass the additional properties through to the
   // ListItem as per the material-ui documentation
   /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
```

Comment:
Do we still need this eslint disable (since we no longer specify `Link as any`)? Can probably remove this as well as the comment describing the rationale for it.

Reply:
@louise-davies I removed the comments in https://github.com/ral-facilities/scigateway/pull/132/commits/44dd5e733d41fd55e73ef0c5ddc53df71c80596c.